app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminCheckProductOnProductGridActionGroup.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>	
+<!--	
+ /**	
+  * Copyright © Magento, Inc. All rights reserved.	
+  * See COPYING.txt for license details.	
+  */	
+-->	
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"	
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">	
+    <actionGroup name="AdminCheckProductOnProductGridActionGroup">	
+        <annotations>	
+            <description>Check the checkbox for the product on the Product Grid</description>	
+        </annotations>	
+        <arguments>	
+            <argument name="product" type="entity"/>	
+        </arguments>	
+
+      <checkOption selector="{{AdminProductGridSection.productRowCheckboxBySku(product.sku)}}" stepKey="selectProduct"/>    	
+
+    </actionGroup>	
+</actionGroups>
```

Comment:
Hi @AnnaAPak. Could I ask you to add an empty line at the end of the file, please?

Reply:
Added, thanks.